test/correctness/simd_op_check.cpp

```diff
@@ -41,6 +41,26 @@ int my_process_id = 0;
 // width and height of test images
 const int W = 256*3, H = 100;
 
+bool can_run_code() {
+
+  // If we can (target matches host), run the error checking Func.
```

Comment:
Especially when committing code to master, we need to follow the convention of the rest of the code. This looks like LLVM's conventions, but among other things, Halide uses 4 spaces instead of 2 for indendation.

I know the Hexagon branch already has this problem, but I've been ignoring it for now.